runner.sh

```diff
@@ -6,25 +6,18 @@ user=nsls2
 image="condaforge"
```

Comment:
I'd suggest naming this image as a part of the env_name value below, i.e. `nsls2-analysis`, so we can update it in the future, but do not register a separate version each time we update the version.

This should eventually be handled by the configuration and templates.

Reply:
Okay done.